test/contrib/isotp_soft_socket.uts

```diff
@@ -58,6 +58,18 @@ assert sniffed[0]['ISOTP'].rx_ext_address == 0xEA
 
 + ISOTPSoftSocket tests
 
+= CAN socket FD
+~ needs_root linux vcan_socket
+
+with ISOTPSoftSocket("can0", tx_id=0x641, rx_id=0x241, fd=True) as s:
```

Comment:
The unit test machine doesn't have "can0". You need to add the interface mockup before you place your code. 
https://github.com/secdev/scapy/blob/363d3766f53c3d55e92b0d51c5cdde7185733e3b/test/contrib/isotp_native_socket.uts#L7C1-L11C1

Ideally, you reuse one of these variables as interface definition, so that the code is consistent.

https://github.com/secdev/scapy/blob/363d3766f53c3d55e92b0d51c5cdde7185733e3b/test/contrib/automotive/interface_mockup.py#L26-L27